src/containers/permissionssummary/components/AllPermissions.js

```diff
@@ -33,12 +35,15 @@ class AllPermissions extends React.Component {
   }
 
   componentDidMount() {
+    this.props.actions.fetchOrganizationsRequest();
```

Comment:
Add this to `static propTypes = { ... }` above. While you're doing that, change all instances of `React.PropTypes` to just `PropTypes`.